### PR TITLE
Garbage collect LB resources for GCE

### PR DESCRIPTION
**Pull request by anfernee**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**: 
To clean up orphan LB resources in GCE

```release-note
NONE
```

### CODE REVIEW

**Comment by anfernee** [comment]
@rramkumar1 Still WIP, 'cause I think I miss some resources in it.

File: cluster/gce/delete-stranded-load-balancers.sh [hunk 1]
```diff
@@ -22,14 +22,31 @@ LIST=$(gcloud --project=${PROJECT} compute target-pools list --format='value(nam
...
 for x in ${LIST}; do
-    if ! gcloud compute --project=${PROJECT} target-pools get-health "${x}" --region=${REGION} 2>/dev/null >/dev/null; then
-	echo DELETING "${x}"
-	gcloud compute --project=${PROJECT} firewall-rules delete "k8s-fw-${x}" -q
-	gcloud compute --project=${PROJECT} forwarding-rules delete "${x}" --region=${REGION} -q
-	gcloud compute --project=${PROJECT} addresses delete "${x}" --region=${REGION} -q
-	gcloud compute --project=${PROJECT} target-pools delete "${x}" --region=${REGION} -q
-        result=1
-    fi
+  if ! gcloud compute --project=${PROJECT} target-pools get-health "${x}" --region=${REGION} 2>/dev/null >/dev/null; then
```
Comment (rramkumar1) [inline cluster/gce/delete-stranded-load-balancers.sh]: Can you add some comments here? Its hard to follow the logic.

Comment (anfernee) [reply cluster/gce/delete-stranded-load-balancers.sh]: added comments.

File: cluster/gce/delete-stranded-load-balancers.sh [hunk 2]
```diff
@@ -14,22 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# A utility for deleting target pools and forwarding rules that are unattached to VMs
+# A utility for deleting stranded load balancer resources in GCE
 PROJECT=${PROJECT:-kubernetes-jenkins}
 REGION=${REGION:-us-central1}
 
+# Deleting external load balancer resources
```
Comment (rramkumar1) [inline cluster/gce/delete-stranded-load-balancers.sh]: This is external L4 right?

File: cluster/gce/delete-stranded-load-balancers.sh [hunk 3]
```diff
@@ -14,22 +14,37 @@
...
+  if ! gcloud compute --project=${PROJECT} target-pools get-health "${x}" --region=${REGION} 2>/dev/null >/dev/null; then
+    echo DELETING LB "${x}"
+    gcloud compute --project=${PROJECT} firewall-rules delete "k8s-fw-${x}" -q
+    gcloud compute --project=${PROJECT} forwarding-rules delete "${x}" --region=${REGION} -q
+    gcloud compute --project=${PROJECT} addresses delete "${x}" --region=${REGION} -q
+    gcloud compute --project=${PROJECT} target-pools delete "${x}" --region=${REGION} -q
+  fi
+done
+
+# Deleting internal load balancer resources
```
Comment (rramkumar1) [inline cluster/gce/delete-stranded-load-balancers.sh]: This is external L7 right?

Comment (anfernee) [reply cluster/gce/delete-stranded-load-balancers.sh]: It's an internal L4.

File: cluster/gce/delete-stranded-load-balancers.sh (hunk 2 above)
Comment (anfernee) [reply cluster/gce/delete-stranded-load-balancers.sh]: yes. basically it's the gce resources created when user creates an external k8s LB.

**Comment by anfernee** [comment]
cc @bowei

**Comment by bowei** [comment]
What is the change being made here? It's not clear from the desc. The original code already was deleting LB resources.
